inspection/background.js

```diff
@@ -126,23 +120,21 @@ function loadListener(url) {
  */
 function createOpenURLEntry(url) {
 	if (url) {
-		chrome.tabs.create({url: url}, function (tab) {
+		chrome.tabs.create({ url: url }, function (tab) {
 			printLog('log', 'OPEN NEW PAGE');
-			localStore?.set({inspectingTab: JSON.parse(JSON.stringify(tab))}, () => {});
-			// todo: is `inspectingTab` a global var or local?
+			localStore?.set({ inspectingTab: JSON.parse(JSON.stringify(tab)) }, () => { });
 			printLog(inspectingTab);
 			updateBadge();
 			loadListener(url);
 		});
 	} else {
-		chrome.tabs.query({active: true, lastFocusedWindow: true}, (tabs) => {
+		chrome.tabs.query({ active: true, lastFocusedWindow: true }, (tabs) => {
 			if (!tabs || tabs.length < 1) {
 				return;
 			}
 			printLog('log', 'CURRENT PAGE');
-			// todo: is `inspectingTab` a global var or local?
 			inspectingTab = JSON.parse(JSON.stringify(tabs[0]));
```

Comment:
now we've removed `inspectingTab`, this line should also be removed? 
or combine line 136 and 137?

Reply:
Done with this change and pushed it to remote